debugdialog.html

```diff
@@ -193,6 +193,9 @@
             break;
         case 'getFeatureVariable':
             response = optimizelyClientInstance.getFeatureVariable(key, variable, userid, attributes)
+            if (typeof response === 'object') {
```

Comment:
Here I would check for variable type instead, but I couldn't figure out how to get variable type in this case. Any advice @thomaszurkan-optimizely?

Reply:
https://docs.developers.optimizely.com/full-stack/docs/optimizelyconfig-javascript